pythonforandroid/recipes/levenshtein/__init__.py

```diff
@@ -0,0 +1,21 @@
+from pythonforandroid.toolchain import CompiledComponentsPythonRecipe
+from os.path import dirname
+
+class LevenshteinRecipe(CompiledComponentsPythonRecipe):
+    name="levenshtein"
+    version = '0.12.0'
+    url = 'https://pypi.python.org/packages/source/p/python-Levenshtein/python-Levenshtein-{version}.tar.gz'
+    depends = [('python2', 'python3'), 'setuptools']
+
+    call_hostpython_via_targetpython = False
+
+    def get_recipe_env(self, arch):
+        env = super(LevenshteinRecipe, self).get_recipe_env(arch)
+        libxslt_recipe = Recipe.get_recipe('libxslt', self.ctx)
```

Comment:
My testing application compiles until these lines get referenced, which I believe may be because the class `Recipe` is called instead of the instance.  But something seems to have accidentally filled my VM up to the brim so I need to clean it out before I can continue testing, so I haven't been able to test the change into an instance.